source/guide_rustdesk.rst

```diff
@@ -40,6 +40,7 @@ Prerequisites
 To make the application accessible from the outside, open three `ports in the firewall <firewall_>`_:
 
 .. code-block:: console
+ :emphasize-lines: 1,3,5
```

Comment:
These lines are highlighted in “includes/open-port.rst” so they should also be highlighted here.